openfisca_us/variables/household/demographic/geographic/county_zip_code.py

```diff
@@ -0,0 +1,12 @@
+from openfisca_us.model_api import *
+
+
+class county_zip_code(Variable):
+    value_type = str
+    entity = Household
+    label = "County code from zip code"
```

Comment:
Is this standard across the country? Could you add a reference?

Reply:
Probably a bit more involved for other states - added a reference